File: back-tussam/src/main/kotlin/com/backtussam/model/Match.kt
package com.backtussam.model

data class Match (
    val id: Int,
    val id_player1: Int? = null,
    val id_player2:  Int? = null,
    val id_player3:  Int? = null,
    val id_player4:  Int? = null,
    val scoreSet1A:  Int = 0,
    val scoreSet1B:  Int = 0,
    val scoreSet2A:  Int = 0,
    val scoreSet2B:  Int = 0,
    val scoreSet3A:  Int = 0,
    val scoreSet3B:  Int = 0,
    val matchResult: String? = null,
    val date: String? = null,
    val open: Boolean = false,
    val type: String? = null,
    val level: Int? = null,
    val confirmResult1: Boolean = false,
    val confirmResult2: Boolean = false,
    val chat:  Int? = null,
    val court:  Int? = null
)

File: front-tussam/src/app/interfaces/match.ts
export interface Match {
    id: number;
    id_player1: number | null;
    id_player2: number | null;
    id_player3: number | null;
    id_player4: number | null;
    scoreSet1A: number | null;
    scoreSet1B: number | null;
    scoreSet2A: number | null;
    scoreSet2B: number | null;
    scoreSet3A: number | null;
    scoreSet3B: number | null;
    matchResult: string | null;
    date: string | null;
    open: boolean;
    type: string | null;
    level: number | null;
    confirmResult1: boolean;
    confirmResult2: boolean;
    chat: number | null;
    court: number | null;
  }
